basetest.pm

```diff
@@ -387,6 +387,7 @@ sub runtest {
         $self->run_post_fail("test $name failed");
     }
 
+    $self->{details}->{execution_time} = time - $starttime . ' s';
```

Comment:
Just a minor thing: I would rather calculate the elapsed time once, and then use it here and two lines later.
So it is ensured it shows the exact same number of seconds.